examples/computer_vision/cifar10_pytorch_inference/model_def.py

```diff
@@ -0,0 +1,95 @@
+"""
+Perform inference on pretrained CIFAR10 from https://github.com/huyvnphan/PyTorch_CIFAR10
+"""
+
+import tempfile
+from typing import Any, Dict, Sequence, Tuple, Union, cast
+
+import torch
+import torchvision
+from torch import nn
+from torchvision import transforms
+import torchvision.models as models
+
+from determined.pytorch import DataLoader, PyTorchTrial, PyTorchTrialContext
+import resnet
+
+# Constants about the data set.
+IMAGE_SIZE = 32
+NUM_CHANNELS = 3
+NUM_CLASSES = 10
+
+TorchData = Union[Dict[str, torch.Tensor], Sequence[torch.Tensor], torch.Tensor]
+
+
+def accuracy_rate(predictions: torch.Tensor, labels: torch.Tensor) -> float:
+    """Return the accuracy rate based on dense predictions and sparse labels."""
+    assert len(predictions) == len(labels), "Predictions and labels must have the same length."
+    assert len(labels.shape) == 1, "Labels must be a column vector."
+
+    return (  # type: ignore
+        float((predictions.argmax(1) == labels.to(torch.long)).sum()) / predictions.shape[0]
+    )
+
+class CIFARTrial(PyTorchTrial):
+    def __init__(self, context: PyTorchTrialContext) -> None:
+        self.context = context
+        self.download_directory = tempfile.mkdtemp()
+
+        # TODO: Load your trained model
```

Comment:
let's add a comment here about ways you can load a model in determined.  e.g. load checkpoint from model registry, load checkpoint from previous experiment, or specifying uuid in experiment config